cmd/crank/render/render.go

```diff
@@ -374,9 +374,9 @@ func Render(ctx context.Context, log logging.Logger, in Inputs) (Outputs, error)
 	xr.SetName(in.CompositeResource.GetName())
 
 	xrCond := xpv1.Available()
-	if d.GetComposite().Ready == fnv1.Ready_READY_FALSE {
+	if d.GetComposite().GetReady() == fnv1.Ready_READY_FALSE {
 		xrCond = xpv1.Creating()
-	} else if d.GetComposite().Ready == fnv1.Ready_READY_UNSPECIFIED && len(unready) > 0 {
+	} else if d.GetComposite().GetReady() == fnv1.Ready_READY_UNSPECIFIED && len(unready) > 0 {
```

Comment:
i did not make this change manually, it came in from running `earthly +reviewable` and/or `earthly +lint`. I actually think it will fix, so that's nice.

* https://github.com/crossplane/crossplane/issues/6680 

Wonder why that didn't get picked up already by the linter runs in https://github.com/crossplane/crossplane/pull/6662 🤔

Reply:
Nice. I'm surprised I missed that and didn't use `GetReady()` myself.